assets/js/base/components/pagination/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Label from '../label';
+import { getIndexes } from './utils.js';
+import './style.scss';
+
+const Pagination = ( { currentPage, pagesToDisplay, onPageChange, totalPages } ) => {
```

Comment:
This could probably use a docblock, or the props.

Reply:
Done in b860620f4c02b6aa8d321ada3f502a7ab73ce95e. :+1: